figures/serializers.py

```diff
@@ -740,12 +741,19 @@ class Meta:
         model = get_user_model()
         editable = False
         fields = (
-            'id', 'username', 'name', 'email', 'country', 'is_active', 'course_activity_date',
-            'year_of_birth', 'level_of_education', 'gender', 'date_joined', 'last_login',
-            'bio', 'courses', 'language_proficiencies', 'profile_image',
+            'id', 'username', 'name', 'email', 'country', 'is_active',
```

Comment:
Please  add new field at the end of the list to reduce git diff

Reply:
Done.